### PR TITLE
improve api-docs

### PR DESCRIPTION
**Pull request by maksymmateiuk**
/accounts/{accountId}/subscriptions /accounts/{accountId}/subscriptions/{subscriptionid} /accounts/{accountId}/tnoptions /accounts/{accountId}/tnoptions/{orderid} /accounts/{accountId}/tnoptions/{orderid}/history /accounts/{accountId}/tnreservation /accounts/{accountId}/tnreservation/{reservationid} /accounts/{accountId}/tollFreePortingValidations /accounts/{accountId}/tollFreePortingValidations/{orderId} /accounts/{accountId}/tnPortoutPasscodes

**⚠️Please do not make any changes to spec files in this repository!⚠️**

**All spec changes should be done in the [api-specs](https://github.com/Bandwidth/api-specs) repository.**

**Please follow the [Guide](https://bandwidth-jira.atlassian.net/wiki/spaces/DX/pages/4098359409/How+to+Update+API+Specifications+and+Contribute+to+Developer+Documentation) for contributing to our developer documentation.**

### CODE REVIEW

File: site/specs/numbers.json [hunk 1]
```diff
@@ -15375,41 +15533,53 @@
...
-            "example": null
+              "type": "string"
+            }
           }
         ],
         "requestBody": {},
         "responses": {
           "200": {
-            "description": "<p>A subscrption as been retrieved. The information is returned.</p>\n",
+            "description": "A subscription as been retrieved. The information is returned.",
```
Comment (mrmeyers99) [inline site/specs/numbers.json:15564]:
```suggestion
            "description": "A subscription has been retrieved. The information is returned.",
```

File: site/specs/numbers.json [hunk 2]
```diff
@@ -15546,125 +15717,113 @@
...
-              "type": "string",
-              "default": null
+              "type": "string"
             },
             "example": "9199918388"
           },
           {
             "name": "customerOrderId",
-            "description": "<p><b class=\"required\">*</b> The Customer Order ID is an ID assigned by the account owner to provide a reference number for the TN Option order.</p>\n",
+            "description": "<b>*</b> The Customer Order ID is an ID assigned by the account owner to provide a reference number for the TN Option order.",
```
Comment (mrmeyers99) [inline site/specs/numbers.json:15730]: let's remove the asterisk and marked "required": true if it's required

File: site/specs/numbers.json [hunk 3]
```diff
@@ -15546,125 +15717,113 @@
...
-              "type": "string",
-              "default": null
+              "type": "string"
             },
             "example": "smckinnon"
           },
           {
             "name": "orderIdFragment",
-            "description": "<p><b class=\"required\">*</b> This search parameter represents the internal Bandwidth Dashboard API ID that has been assigned to the TN Option Order.  This parameter is the first few characters of the internal ID - the entire ID does not need to be specified.</p>\n",
+            "description": "<b>*</b> This search parameter represents the internal Bandwidth Dashboard API ID that has been assigned to the TN Option Order. This parameter is the first few characters of the internal ID - the entire ID does not need to be specified.",
```
Comment (mrmeyers99) [inline site/specs/numbers.json:15800]: let's remove the asterisk and marked "required": true if it's required

File: site/specs/numbers.json [hunk 4]
```diff
@@ -16325,54 +16530,52 @@
           "/accounts"
         ],
         "description": "<p>Retrieve information about one or more Telephone Numbers (TNs), where the TNs are specified in POST body.</p>\n",
-        "operationId": "POST /accounts/{accountId}/tnPortoutPasscodes",
+        "operationId": "Retrieve tn port-out",
```
Comment (mrmeyers99) [inline site/specs/numbers.json:16533]: should it say `Retrieve tn port-out passcodes`?

File: site/specs/numbers.json [hunk 5]
```diff
@@ -15679,68 +15838,66 @@
...
                 }
               }
             }
           }
         },
         "responses": {
           "201": {
-            "description": "<p>Created <br> A 201 response indicates that an order has been created.</p>\n",
+            "description": "Created A 201 response indicates that an order has been created.",
```
Comment (ajrice6713) [inline site/specs/numbers.json:15871]:
```suggestion
            "description": "Created",
```
Just `Created` is fine

File: site/specs/numbers.json [hunk 6]
```diff
@@ -15679,68 +15838,66 @@
...
-                    "value": "<?xml version=\"1.0\" encoding=\"UTF-8\" standalone=\"yes\"?>\n<TnOptionOrderResponse>\n <ResponseStatus>\n <ErrorCode>5081</ErrorCode>\n <Description>Number Format 'wrong' is invalid.</Description>\n </ResponseStatus>\n </TnOptionOrderResponse>\n"
+                    "value": "<?xml version=\"1.0\" encoding=\"UTF-8\" standalone=\"yes\"?>\n<TnOptionOrderResponse>\n  <ResponseStatus>\n    <ErrorCode>5081</ErrorCode>\n    <Description>Number Format 'wrong' is invalid.</Description>\n  </ResponseStatus>\n</TnOptionOrderResponse>"
                   }
                 }
               }
             }
           },
           "409": {
-            "description": "<p>Conflict <br> A 409 response indicates that order could not be created.\nError text and an error code will be provided in the ErrorList element.</p>\n",
+            "description": "Conflict A 409 response indicates that order could not be created. Error text and an error code will be provided in the ErrorList element.",
```
Comment (ajrice6713) [inline site/specs/numbers.json:15895]:
```suggestion
            "description": "Conflict. Error text and an error code will be provided in the ErrorList element.",
```

File: site/specs/numbers.json [hunk 7]
```diff
@@ -15754,55 +15911,53 @@
...
                   }
                 }
               }
             }
           },
           "404": {
-            "description": "<p>NOT FOUND - the order id does not exist in the system</p>\n",
+            "description": "NOT FOUND - the order id does not exist in the system.",
```
Comment (ajrice6713) [inline site/specs/numbers.json:15955]:
```suggestion
            "description": "Not Found. The order id does not exist in the system.",
```

File: site/specs/numbers.json [hunk 8]
```diff
@@ -15816,59 +15971,48 @@
...
-              "application/xml": {
-                "examples": {
-                  "application/xml": {
-                    "description": "application/xml",
-                    "value": null
-                  }
-                }
-              }
-            }
+            "description": "NOT FOUND - the order id does not exist in the system."
```
Comment (ajrice6713) [inline site/specs/numbers.json:16015]:
```suggestion
            "description": "Not Found. The order id does not exist in the system."
```

**Review by ajrice6713 — CHANGES_REQUESTED** [review]

File: site/specs/numbers.json (hunk 3 above)
Comment (maksymmateiuk) [reply site/specs/numbers.json:15800]: It's not a required field. It's a specific rule for that query.  ```If no date range or specific query parameter (marked by <b>*</b> below) is provided, the order results will be limited to the last two years.```

**Review by ajrice6713 — APPROVED** [review]

File: site/specs/numbers.json (hunk 3 above)
Comment (mrmeyers99) [reply site/specs/numbers.json:15800]: ahh

**Review by mrmeyers99 — APPROVED** [review]

**Pull request merged**